### PR TITLE
Use upstream PHP FPM Alpine Docker image

### PR DESCRIPTION
**Pull request by J0WI**
This switches the base to the upstream PHP FPM Alpine Docker image, so we can benefit from the maintenance there.
I included and configured the same php extensions as before.

This might be a dependency task of #56.

### CODE REVIEW

**Comment by nijel** [comment]
It fails to test as the test setup relies on using more recent apline (the PHP images are based on 3.4, while we were on 3.6). 

That also pops up question whether it's better to stick with maintained alpine image having current all libs or maintained PHP image which has current PHP, but everything else is older. Do you know if Alpine 3.4 still receives security fixes?

**Comment by J0WI** [comment]
I don't know why the tests are failing. Both versions build and run.
Can you point me to the relevant lines?

**Comment by nijel** [comment]
I don't see anything obvious, but apparently the container behaves differently than the original one.

The problem can be seen [here](https://travis-ci.org/phpmyadmin/docker/jobs/324381951#L1965) - the test script is not found. Maybe the working directory is different than it used to be and that confuses the script lookup:

https://github.com/phpmyadmin/docker/blob/f5644c53c8cf05060aedcf1e1906a32bb355a513/testing/test-docker.sh#L72-L76

**Comment by J0WI** [comment]
Thanks, I've updated the path in the tests.

**Comment by J0WI** [comment]
@nijel would be great to get some feedback here.

> Do you know if Alpine 3.4 still receives security fixes?

https://github.com/docker-library/php/issues/504#issuecomment-359111672

File: testing/test-docker.sh [hunk 1]
```diff
@@ -69,8 +69,8 @@ done
 
 # Perform tests
 if [ $ret -eq 0 ] ; then
-    if [ -f ./phpmyadmin_test.py ] ; then
-        FILENAME=./phpmyadmin_test.py
+    if [ -f /phpmyadmin_test.py ] ; then
+        FILENAME=/phpmyadmin_test.py
```
Comment (nijel) [inline testing/test-docker.sh]: Probably both variants should stay to allow local testing.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

I've applied following fixups on top of your changes:

* d53771b Bring back original SQL lookup
* 3bd49fb Avoid using tabs

**Pull request merged**